### ISSUE TITLE
Client admin accounts and temp links for file sharing

### ISSUE THREAD

**Issue by os-rsingleton** [projectsend/projectsend#660]
[X] Feature request

**Feature 1:**
It would be nice to have the ability to create a client admin account that would act as a parent account to which child accounts would belong.
The client admin would be able to:

- and manage client accounts marked as belonging to the parent account.
- add/remove accounts to any of the client groups that are available to the client admin account.
- limit children accounts to upload/download, download-only.
- select which groups all uploaded files belonging to itself or its children are members of.
- specify whether or not a child account can add files to available groups and select which groups the child account can add files to.

Basically the parent account would be the same as a system user with account management privileges except that it can only manage itself and accounts that belong to it and provide resources that are available to it.

**Feature 2:**
The ability to set a temporary link for downloading a file that will allow users/client to share files.

### Environment

* ProjectSend version: **r1053**
* php version: **5.6.30**
* MySQL version: **5.6.41-84.1**

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
